kibana.d.ts

```diff
@@ -20,7 +20,9 @@
 /**
  * All exports from TS source files (where the implementation is actually done in TS).
  */
-export * from './target/types/type_exports';
+export * from './src/core/public';
+export * from './src/core/server';
```

Comment:
Should this be something like `export * as CorePublic from './src/core/public'` so that any types with similar names don't conflict with one another?

Reply:
Good point, I've put them in their own namespaces, but kept it to e.g. "Public" as opposed to "CorePublic"